src/app/components/wrapper-components/part-creation/part-creation.component.ts

```diff
@@ -349,31 +352,34 @@ export class PartCreationComponent implements OnInit, OnDestroy {
             Debug.display('PartCreationComponent', 'onCurrentConfigRoomUpdate', 'LAST UPDATE : the game is cancelled');
             return this.onGameCanceled();
         } else {
+            const oldConfigRoom: ConfigRoom | null = this.currentConfigRoom;
```

Comment:
c'pas la best fonction pour dire "voici quels types de changement ça pourrait être, on détermine quel type c'est et on redirige vers this.actAccordingly(bilboul) tiens :/

Reply:
oui, si tu veux proposer un refactor je te laisse proposer un nom de ticket ! (je suis d'accord que le code de cette fonction est dégueu)
Mais c'est hors scope, ici c'est un ticket de bugfix

Reply:
"refactor on part-creation", ce ticket a deux fichier modifié, c'est in-scope !

Reply:
ce ticket est "bugfix", pas "refactor" ! Si je voulais faire un refactor de part-creation, j'aurais pas fait ce ticket maintenant :p

Reply:
Ok mais, on sais rajouter un test qui est bien foiré avant la modif et fixé après la modif ?
Clean TDD et long terme vie ?